oh-my-bash.sh

```diff
@@ -144,6 +144,9 @@ else
       source "$OSH_CUSTOM/$OSH_THEME/$OSH_THEME.theme.sh"
     elif [ -f "$OSH_CUSTOM/themes/$OSH_THEME/$OSH_THEME.theme.sh" ]; then
       source "$OSH_CUSTOM/themes/$OSH_THEME/$OSH_THEME.theme.sh"
+		# https://github.com/ohmybash/oh-my-bash/issues/70
```

Comment:
@Kreyren I prefer we keep only one naming convention for the theme or plugins. If agnoster theme file name is `agnostic-bash.theme.sh`, then we should refactor it to match with `$OSH_THEME`.theme.sh.

Cheers,
Toan

Reply:
Fixed in https://github.com/ohmybash/oh-my-bash/pull/71/commits/fb6e15a38a079597787bae2644efe025f5a90029